platform-sdk/swirlds-common/src/main/java/com/swirlds/common/system/SwirldState.java

```diff
@@ -65,24 +54,27 @@ default void init(
     }
 
     /**
-     * Given a round of consensus ordered events, update the state to reflect their effect. Events are iterated in
-     * consensus order by the {@link Round#iterator()}. Transactions in each event are iterated in consensus order by
-     * the {@link ConsensusEvent#consensusTransactionIterator()}. Transactions in a single event occur after all the
-     * transactions in the previous event and before all the transactions in the next event.
-     *
+     * Provides the application an opportunity to perform operations on transactions in an event prior to handling.
+     * Called against a given {@link Event} only once, globally (not once per state instance) This method may modify the
+     * {@link Transaction}s in the event by doing nothing, adding additional signatures, removing existing signatures,
+     * replacing signatures with versions that expand the public key from an application specific identifier to an
+     * actual public key, or attaching metadata. Additional signatures extracted from the transaction payload can also
+     * be added to the list of signatures to be verified.
```

Comment:
Some documentation for `preHandle()` was lost here. Check what was on the `SwirldState2.prehandle()` method before